app/views/layouts/_footer.html.slim

```diff
@@ -9,7 +9,8 @@ footer.footer style="background-color: rgba(223, 223, 255, 0.9);"
       .credits.ml-auto
         span.copyright
           | Do Art Daily 
-          = link_to PatchNote.last.patch, "/news"
+          - unless PatchNote.last.nil?
```

Comment:
I should probably add a comment explaining the purpose of this change: it's because the patch note is not present in the test environment, and otherwise this bit of code breaks *every single view*. I don't really know how patch notes work, but maybe it'd be worth looking into an alternative that can be generated from an e.g. `patchnotes.yml` without having specifically to run a task at all. Though AFAICT you haven't been using the patch notes system or bumping the version number very much recently anyway.

Reply:
Yeah, this is being overhauled. Expect patchnotes to be dumped in DAD 2.0.